examples/programrequests/ProgramValidation.json

```diff
@@ -0,0 +1,61 @@
+{
+  "version": "1.0.0",
+  "messageType": "PROGRAM_VALIDATION",
+  "validationRequests": [
+    {
+      "key": "customField",
+      "queryResult": {
+        "users": {
+          "totalCount": 2
+        }
+      }
+    },
+    {
+      "key": "customFieldFormat",
+      "queryResult": {
+        "users": {
+          "count": 2,
+          "data": [
+            {
+              "customFields": {
+                "lastPurchaseDate": 1555975846849,
+                "birthday": "--01-01",
+                "vip_totalValue": 9251,
```

Comment:
What is this field for?

Reply:
@rebeccachan  These are just examples for testing, not actual schema files. So for the most part they can be ignored.